src/Components/App.jsx

```diff
@@ -1,23 +1,15 @@
 import React from "react";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { BrowserRouter as Router } from "react-router-dom";
 import { Provider } from "react-redux";
 import store from "../Redux/store";
 
 import "./App.scss";
-import NavBar from "./Shared/NavBar/NavBar.jsx";
-import Home from "./Home/Home.jsx";
-import SignIn from "./SignIn/SignIn.jsx";
+import Routes from "./Routes";
 
 const App = () => (
   <Provider store={store}>
     <Router>
-      <div>
-        <NavBar />
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route exact path="/signin" component={SignIn} />
-        </Switch>
-      </div>
+      <Routes />
```

Comment:
You should put back the routes here instead of importing them from a separate file.